nextcord/partial_emoji.py

```diff
@@ -113,6 +115,13 @@ def from_dict(cls: Type[PE], data: Union[PartialEmojiPayload, Dict[str, Any]]) -
             name=data.get("name") or "",
         )
 
+    @classmethod
+    def from_reaction(cls: Type[PE], data: DefaultReaction) -> PE:
```

Comment:
This method name could be more descriptive, maybe something like `from_default_reaction`